style.css

```diff
@@ -437,3 +468,332 @@ textarea {
   font-size: 17px;
   line-height: 24px;
 }
+
+/* Desktop site media querry starts here */
+@media only screen and (min-width: 992px) {
+  #nameLogo {
+    margin-left: 8rem;
+  }
+
```

Comment:
- I'd like to kindly suggest that your min-width property should be in the range of 768px or wider) because the min-width you are currently using has caused your website to appear differently on tablet screen (768px) but perfectly on screens above 768px.

Here is what I mean 👇🏽 

Here is what I can see compared to what's on Figma 👇🏽 
| Your header |
| :---:   | 
|<img width="582" alt="Screenshot 2023-01-20 at 10 47 50" src="https://user-images.githubusercontent.com/99246359/213756456-9288afa2-04dd-41b3-88aa-87db70b4d6d5.png">  |

|some part of your project card and about section |
| :---:   | 
|<img width="588" alt="Screenshot 2023-01-20 at 10 48 13" src="https://user-images.githubusercontent.com/99246359/213756496-3b253a02-2a0f-4b89-bd99-3c5dc0b09554.png">  |

|Your contact form |
| :---:   | 
|<img width="580" alt="Screenshot 2023-01-20 at 10 50 52" src="https://user-images.githubusercontent.com/99246359/213756523-ba688606-c2f7-4d44-be10-03a419801c2b.png">  |

⚠️ Kindly note that all sections of your portfolio website are supposed to contain the entire screen, so if you use min-width: 768px, it will work perfectly. 🌸 

What do you think about implementing all of these?

Reply:
thank you ery mush for the suggestions